package.json

```diff
@@ -14,12 +14,14 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@babel/core": "^7.4.0",
-    "@babel/parser": "^7.4.2",
+    "@babel/core": "^7.13.8",
+    "@babel/generator": "^7.13.0",
+    "@babel/parser": "^7.13.4",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.4.4",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@babel/plugin-syntax-optional-chaining": "^7.2.0",
     "@babel/preset-flow": "^7.0.0",
+    "@babel/traverse": "^7.13.0",
```

Comment:
I've included this here just incase the vendored version needs a sub-dependency that isn't provided by something else.